en/docs/deploy/deploy-a-containerized-choreo-component.md

```diff
@@ -46,9 +46,19 @@ Let's add a containerized REST API component by following these steps:
 
 6. Click **Authorize with GitHub** to allow the REST API to access your GitHub account.
 
+    If you have not already authorized Choreo applications, click **Authorize Choreo Apps** when prompted. 
     Once you perform the authorization, the **GitHub Account** field in the **Connect Repository** dialog box displays the GitHub account you authorized the REST API to access. 
 
-8. Select the following values to connect the repository.
+
+7. Select the following values to connect the repository:
+
+    !!! info
+        To successfully build your container with Choreo, it is essential that you explicitly define a User ID (`UID`) under the `USER` instruction in your Dockerfile. You can refer to the [sample Dockerfile](https://github.com/wso2/choreo-sample-apps/blob/main/go/rest-api/Dockerfile) for guidance.
```

Comment:
Suggestion:
To successfully build your container with Choreo, it is essential to explicitly define a User ID (`UID`) in the `USER` instruction of your Dockerfile. For guidance, refer to the [sample Dockerfile](https://github.com/wso2/choreo-sample-apps/blob/main/go/rest-api/Dockerfile).